srcpkgs/sagemath/template

```diff
@@ -26,14 +26,14 @@ depends="eclib-devel fflas-ffpack flintlib-devel gcc-fortran gd-devel
  python3-sympy python3-traitlets sage-data-combinatorial_designs
  sage-data-conway_polynomials sage-data-elliptic_curves sage-data-graphs
  sage-data-polytopes_db sympow tachyon threejs-sage"
-checkdepends="$depends pythran python3-Sphinx meson"
+checkdepends="$depends pythran python3-Sphinx"
 short_desc="Open source mathematics software"
 maintainer="Gonzalo Tornaría <tornaria@cmat.edu.uy>"
 license="GPL-2.0-or-later"
 homepage="https://www.sagemath.org/"
 changelog="https://github.com/sagemath/sage/releases"
 distfiles="https://github.com/sagemath/sage/archive/refs/tags/$version.tar.gz"
-checksum=e7125f13495e1068edab73735aca7f9b2c655688096e9d109e628c023e76411f
+checksum=3f72b8d13d8bb472daaeabff4feba98e392028328e20012980da9c457dc49fb0
 nocross="due to ntl (eclib), fflas-ffpack, givaro, linbox, sympow, maxima"
```

Comment:
Can you add `singular` here, just so we don't forget?
`nocross="due to ntl (eclib, singular), fflas-ffpack, givaro, linbox, sympow, maxima"`

Reply:
But singular cross builds just fine, doesn't it? Is the cross-built singular not suitable to build sagemath?

Reply:
indeed, the cross-built singular (i.e. built without ntl) gives a lot of segfaults in sage doctests.

Reply:
yes, i agree.
sounds good.